client/naturalcrit/loginPage/loginPage.jsx

```diff
@@ -35,8 +37,16 @@ const LoginPage = React.createClass({
 		window.document.onkeypress = (e)=>{
 			if(e.code == 'Enter') this.handleClick();
 		}
+
+		this.handleRedirectURL();
 	},
 
+	handleRedirectURL : function() {
+		if(!this.props.redirect) { 
+			return window.localStorage.removeItem(RedirectLocation);
+		};
+		return window.localStorage.setItem(RedirectLocation, this.props.redirect);
+	},
```

Comment:
Change this to sessionStorage, as the nature of the redirection is temporary.

Reply:
Updated to sessionStorage in latest commit